Makefile

```diff
@@ -235,13 +235,19 @@ test-config: securedrop/config.py
 .PHONY: dev
 dev:  ## Run the development server in a Docker container.
 	@echo "███ Starting development server..."
-	@OFFSET_PORTS='false' DOCKER_BUILD_VERBOSE='true' $(DEVSHELL) $(SDBIN)/run
+	@OFFSET_PORTS='false' DOCKER_BUILD_VERBOSE='true' SLIM_BUILD=1 $(DEVSHELL) $(SDBIN)/run
 	@echo
 
 .PHONY: dev-tor
 dev-tor:  ## Run the development server with onion services in a Docker container.
-	@echo "███ Starting development server..."
-	@OFFSET_PORTS='false' DOCKER_BUILD_VERBOSE='true' USE_TOR='true' $(DEVSHELL) $(SDBIN)/run
+	@echo "███ Starting development server with onion services..."
+	@OFFSET_PORTS='false' DOCKER_BUILD_VERBOSE='true' USE_TOR='true' SLIM_BUILD=1 $(DEVSHELL) $(SDBIN)/run
+	@echo
+
+.PHONY: dev-full
```

Comment:
Do we need a `dev-full` target? We do need the full build for automated testing purposes, but it doesn't seem like there are development tasks that would require it. (Unless folks are VNCing in to the container to run TBB there instead of locally, which seems unnecessary.)

Reply:
I don't have any use case for it, can drop it and we can re-add it later if one does come up.